src/overlay.cpp

```diff
@@ -689,6 +689,52 @@ void init_system_info(){
       trim(gpu);
       driver = exec("glxinfo | grep 'OpenGL version' | sed 's/^.*: //' | cut -d' ' --output-delimiter=$'\n' -f1- | grep -v '(' | grep -v ')' | tr '\n' ' ' | cut -c 1-");
       trim(driver);
+
+// Get WINE version
+
+      wineProcess = get_exe_path();
+      trim(wineProcess);
```

Comment:
Shouldn't be needed probably. Any whitespace is actually in the file's name then.